pkg/shell/nav.tsx

```diff
@@ -366,6 +412,7 @@
             target: {
                 host: current_machine.address,
                 path: current_machine_manifest_items.items.apps.path,
+                hash: "/",
```

Comment:
This added line is not executed by any test.

Reply:
We could add a test for this. This is about the "Edit" button in the "Applications" section. It only appears when at least one applications is installed.